.automation/build.py

```diff
@@ -361,6 +362,28 @@ def list_descriptors_for_build():
     return descriptors, linters_by_type
 
 
+def get_descriptor_stat():
+    cwd_to_use = os.getcwd() + "/megalinter/descriptors/"
+
+    linters_by_type = {"language": 0, "format": 0, "tooling_format": 0, "other": 0}
+
+    for file_name in glob.iglob(
+        cwd_to_use + "*.megalinter-descriptor.yml", recursive=False
+    ):
+        process = subprocess.run(
+            ["yq", "e", ".descriptor_type", file_name],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
+            universal_newlines=True,
+            cwd=cwd_to_use,
+            shell=False,
+        )
```

Comment:
To be honest, I don't know Python before starting to contribute to this project, so I don't know all eco-system, including your recommendation `PyYAML`
If I used `yq`, this is because I know it and used in past on other projects. Have in mind also that I used it directly through docker in bash script to build the architecture diagram (see https://github.com/megalinter/megalinter/issues/857#issuecomment-974614413)